Art/niyati/index.css

```diff
@@ -0,0 +1,92 @@
+.container {
+  width: 100%;
+  overflow: hidden;
+  background: url(./image/sky.jpg);
```

Comment:
## Missing files
Missing files ( and folder image) in your folder:
- image/sky.jpg
- image/road.jpg
- image/city.png